src/poprox_recommender/default.py

```diff
@@ -245,6 +309,44 @@ def compute_similarity_matrix(todays_article_vectors):
                 similarity_matrix[i, j] = similarity_matrix[j, i] = np.dot(value1, value2)
     return similarity_matrix
 
+def find_topic(past_articles: List[Article], article_id: UUID):
+    # each article might correspond to multiple topic
+    for article in past_articles:
+        if article.article_id == article_id:
+            return [mention.entity.name for mention in article.mentions]
+
+def normalized_topic_count(topic_count_dict):
+
+    normalized_count = {}
+    total_count = sum(topic_count_dict.values())
+    for topic in topic_count_dict:
+        normalized_count[topic] = topic_count_dict[topic] / total_count
```

Comment:
This will work fine as written, but you can save yourself some typing by doing it as a one-liner:
```
normalized_counts = {key: value/total_count for key,value in topic_count_dict.items()}
```